aboutlibraries-core/src/androidMain/kotlin/com/mikepenz/aboutlibraries/util/AndroidParser.kt

```diff
@@ -21,9 +24,9 @@ actual fun parseData(json: String): Result {
         val mappedLicenses = licenses.associateBy { it.hash }
         val libraries = metaData.getJSONArray("libraries").forEachObject {
             val libLicenses = optJSONArray("licenses").forEachString { mappedLicenses[this] }.mapNotNull { it }.toHashSet()
-            val developers = optJSONArray("developers").forEachObject {
+            val developers = optJSONArray("developers")?.forEachObject {
```

Comment:
`forEachObject` will already handle the null case and return an `emptyList` if the `opt` returned null, no need to add the `?` elvis operator

Reply:
Both the native and Android versions return a nullable so it's best to leave it just in case.

#### Android
```Java
@Nullable public JSONArray optJSONArray(@Nullable String name) {
    Object object = opt(name);
    return object instanceof JSONArray ? (JSONArray) object : null;
}
```

#### Native
```kotlin
internal fun JsonObject.optJSONArray(key: String): JsonArray? {
    return get(key)?.jsonArray
}
```

Reply:
Yes, however `forEachObject` will take a nullable and return an `emptyList`